Esignet/scripts/Esignet_Test_Script.jmx

```diff
@@ -2133,12 +2375,8 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
                 <stringProp name="Header.value">application/json</stringProp>
               </elementProp>
               <elementProp name="" elementType="Header">
-                <stringProp name="Header.name">X-XSRF-TOKEN</stringProp>
```

Comment:
In complete flow, Get Csrf Token Endpoint API is missing in OTP flow alone

Reply:
One more subtask, Name presence assertion - this implementation is wrong. We have to check the user name specific for the user that is logged in.